.changeset/cool-poets-provide.md

```diff
@@ -0,0 +1,4 @@
+---
+"@finos/legend-extension-dsl-serializer": patch
```

Comment:
why are we touching `DSL serializer`, this is relational-related so I expect to only see changes in `@finos/legend-graph`